### PR TITLE
[PAPI-252] Vault examples

### PR DESCRIPTION
**Pull request by drmathias**
Examples for all vault requests and responses, as well as non-vault related event log responses..

### CODE REVIEW

File: src/Opdex.Platform.WebApi/Models/Responses/Transactions/TransactionEvents/LiquidityPools/SwapEventResponseModel.cs [hunk 1]
```diff
@@ -2,14 +2,51 @@
...
+        /// <summary>
+        /// Amount of CRS tokens output.
+        /// </summary>
+        /// <example>2.50000000</example>
         public FixedDecimal AmountCrsOut { get; set; }
+
+        /// <summary>
+        /// Amount of SRC tokens output.
+        /// </summary>
+        /// <example>0.000500000000000000</example>
```
Comment (mrtpain) [inline src/Opdex.Platform.WebApi/Models/Responses/Transactions/TransactionEvents/LiquidityPools/SwapEventResponseModel.cs:43]: Nit could make this 0 if you want it to look more like a real example, nbd

File: src/Opdex.Platform.WebApi/Models/Responses/Transactions/TransactionEvents/MiningPools/MineEventResponseModel.cs [hunk 2]
```diff
@@ -2,11 +2,33 @@
...
 {
+    /// <summary>
+    /// Mining event.
+    /// </summary>
     public abstract class MineEventResponseModel : TransactionEventResponseModel
     {
+        /// <summary>
+        /// Address of the miner.
+        /// </summary>
+        /// <example></example>
```
Comment (mrtpain) [inline src/Opdex.Platform.WebApi/Models/Responses/Transactions/TransactionEvents/MiningPools/MineEventResponseModel.cs:13]: Missed a couple examples

**Review by mrtpain — APPROVED** [review]
One or two suggestions, lgtm

File: src/Opdex.Platform.WebApi/Models/Responses/Transactions/TransactionEvents/LiquidityPools/SwapEventResponseModel.cs (hunk 1 above)
Comment (drmathias) [reply src/Opdex.Platform.WebApi/Models/Responses/Transactions/TransactionEvents/LiquidityPools/SwapEventResponseModel.cs]: Good spot 👍

**Pull request merged**